saidump/SaiDump.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#include "swss/table.h"
+#include "meta/sai_serialize.h"
+#include <nlohmann/json.hpp>
+
+namespace syncd
+{
+    class SaiDump
+    {
+        public:
+            SaiDump();
+            ~SaiDump();
+            void handleCmdLine(int argc, char **argv);
+            void dumpFromRedisDb(int argc, char **argv);
+            void printUsage();
+            sai_status_t dumpFromRedisRdbJson();
+            void traverseJson(const nlohmann::json & jsn);
+            void dumpGraphFun(const swss::TableDump& td);
+            void print_attributes(size_t indent, const swss::TableMap& map);
```

Comment:
All functions are named in camelcase follow the schema

Reply:
Fixed.